third-culture-software/bhima
# fix(i18n): fix default database unit translations

Fixes a typo on the Analysis of Auxiliary Cashboxes report.

Closes #6511.

---

To test this:

1. Build the default database (`yarn build:clean`)
2. Set up a new enterprise (put "test" for everything).
3. Log in
4. Go to `Admin > Roles`
5. Edit the permissions of the administrator.
6. Note that all permissions/units are translated.
